build/check_build.py

```diff
@@ -46,26 +37,68 @@ def check_icon_location_pointer_correct(root):
     tree = xml.etree.ElementTree.parse(os.path.join(root, "plugin.xml"))
     root = tree.getroot()
     return len(root.findall(".//property[@value='{}']".format(icon_location))) == 1
+    
+
+# Checks that Test files are following the correct naming convention in ibex_gui\base\
+def check_test_file_format(root):
+    file_named_wrong = False
+    test_files_all = []
+    test_files_failing = []
+    test_exts = ("*Test.java", "Test*.java", "*Tests.java", "*TestCase*.java", "package-info.java")
+
+    #recursively search through the base\ directory for all .java files
+    entries = glob.glob("**/*.java", root_dir=root, recursive=True)
```

Comment:
Don't we want to search only the `.tests` plugins as mentioned in the first acceptance criteria?

```suggestion
    entries = glob.glob("*.tests/**/*.java", root_dir=root, recursive=True)
```